src/main.js

```diff
@@ -91,23 +105,17 @@ window.addEventListener("DOMContentLoaded", () => {
       link.addEventListener("click", accordionHashLinks);
     });
 
-
     // Video
-    let videoThumbnails = document.querySelectorAll('.video-thumbnail');
+    let videoThumbnails = document.querySelectorAll(".video-thumbnail");
```

Comment:
I know its not part of this ticket, but this should be extracted into a initVideo() function.